ginga/util/rgb_cms.py

```diff
@@ -15,10 +15,10 @@
 
 from . import paths
 
+from PIL import Image
 # How about color management (ICC profile) support?
 try:
     import PIL.ImageCms as ImageCms
```

Comment:
So there is still a chance PIL has no CMS or something?

Reply:
Possibly. For a long time, it was always included when installing via `pip`, but was missing when installing via `conda` (just the CMS part). Lately, it has been included even when installing via conda, but I'm not yet assuming that is fully resolved yet.  For most uses of Ginga, CMS is not that important, so we can optionally handle its availability this way.